config/locales/en.yml

```diff
@@ -647,6 +647,10 @@ en:
         role:
           permissions:
             dependency_missing: "need to also include '%{dependency}' as '%{permission}' is selected."
+        time_entry:
+          attributes:
+            hours:
+              day_limit: "more than 24 hours logged for the date."
```

Comment:
I tried a more descriptive error message in https://github.com/opf/openproject/pull/7954/commits/46ff3e593a0724654c742818b1af1c49371cd59c. I deem it good enough now. If you think otherwise (as a note to self) we could look into https://blog.bigbinary.com/2019/04/22/rails-6-allows-to-override-the-activemodel-errors-full_message-format-at-the-model-level-and-at-the-attribute-level.html